pom.xml

```diff
@@ -31,6 +31,12 @@
             <version>RELEASE</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>RELEASE</version>
```

Comment:
When working with maven, it is a good practice to put versions in `properties` section.

In your `properties` tags you can add 

`<lombok.version>RELEASE</lombok.version>` - you can put whatever name you want, even `<lombok-is-the-best.version>RELEASE</lombok-is-the-best.version>`

And then in your lombok dependency you provide:

`<version>${lombok.version}</version>`